src/main/java/htsjdk/samtools/util/IntervalList.java

```diff
@@ -397,12 +398,16 @@ public static IntervalList copyOf(final IntervalList list){
      * @return an IntervalList object that contains the headers and intervals from the file
      */
     public static IntervalList fromFile(final File file) {
-        final BufferedReader reader= IOUtil.openFileForBufferedReading(file);
+        return fromPath(file.toPath());
+    }
+
+    public static IntervalList fromPath(final Path path) {
+        final BufferedReader reader = IOUtil.openFileForBufferedReading(path);
         final IntervalList list = fromReader(reader);
         try {
             reader.close();
         } catch (final IOException e) {
-            throw new SAMException(String.format("Failed to close file %s after reading",file));
```

Comment:
You might want to use the more ugly path.toUri().toString() because the path.toString() tends to drop the scheme.